packages/orbit-components/src/OrbitProvider/ThemeProvider.stories.tsx

```diff
@@ -1,63 +1,12 @@
 import * as React from "react";
-import { select, boolean, object } from "@storybook/addon-knobs";
+import { object } from "@storybook/addon-knobs";
 import { action } from "@storybook/addon-actions";
 
-import useTranslate from "../hooks/useTranslate";
 import Button from "../Button";
 import getTokens from "../getTokens";
-import languages from "../data/dictionary";
-import Badge from "../Badge";
-import Tooltip from "../Tooltip";
 
 import OrbitProvider from ".";
 
-const ButtonWithTranslation = () => {
-  const translate = useTranslate();
-  return <Button>{translate("button_close")}</Button>;
-};
-
-export default {
-  title: "OrbitProvider",
-};
```

Comment:
Please don't delete these lines, otherwise the whole component is gone from Storybook 😄

Reply:
accident :D